src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -89,30 +85,72 @@ public static void AddTodoTemplateSwaggerGen(this IServiceCollection services)
     {
         services.AddSwaggerGen(options =>
         {
-            options.AddSecurityDefinition("bearerAuth", new OpenApiSecurityScheme
-            {
-                Name = "Authorization",
-                Type = SecuritySchemeType.Http,
-                Scheme = "bearer",
-                BearerFormat = "JWT",
-                In = ParameterLocation.Header,
-                Description = "JWT Authorization header using the Bearer scheme."
-            });
+            options.AddSecurityDefinition("bearerAuth",
+                new OpenApiSecurityScheme
+                {
+                    Name = "Authorization",
+                    Type = SecuritySchemeType.Http,
+                    Scheme = "bearer",
+                    BearerFormat = "JWT",
+                    In = ParameterLocation.Header,
+                    Description = "JWT Authorization header using the Bearer scheme."
+                });
 
             options.AddSecurityRequirement(new OpenApiSecurityRequirement
             {
                 {
                     new OpenApiSecurityScheme
                     {
-                        Reference = new OpenApiReference
-                        {
-                            Type = ReferenceType.SecurityScheme,
-                            Id = "bearerAuth"
-                        }
+                        Reference = new OpenApiReference { Type = ReferenceType.SecurityScheme, Id = "bearerAuth" }
                     },
-                    new string[] {}
+                    new string[] { }
                 }
             });
         });
     }
+
+    public static void AddHealthChecks(
+        this IServiceCollection services,
+        IConfiguration configuration,
+        IWebHostEnvironment environment)
+    {
+        var appsettings = configuration.GetSection(nameof(AppSettings)).Get<AppSettings>();
+
+        var healthCheckSetting = appsettings.HealCheckSettings;
+        
+        if (!healthCheckSetting.EnableHealthChecks)
```

Comment:
please do not use `!` operator at the start of a condition, it's not clear in the code. instead use a comparison with `false` like this:

```cs
if (healthCheckSettings.EnableHealthChecks is false)
```